diff --git a/src/main.cpp b/src/main.cpp
index f77c349..0e8702d 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -12,6 +12,7 @@
 #include <QQuickStyle>
 #include <QUrl>
 #include <QtQml>
+#include <QIcon>
 
 #ifdef Q_OS_ANDROID
 #include <QGuiApplication>
@@ -39,6 +40,7 @@ class AbstractDailyWeatherForecast;
 Q_DECL_EXPORT int main(int argc, char *argv[])
 {
     QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+    QIcon::setThemeName(QStringLiteral("breeze"));
 
 #ifdef Q_OS_ANDROID
     QGuiApplication app(argc, argv);
