diff --git a/src/main.cpp b/src/main.cpp
index 022620f..0c1866c 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -15,6 +15,7 @@
 #include <QQuickStyle>
 #include <QUrl>
 #include <QtQml>
+#include <QIcon>
 
 #include <KAboutData>
 #include <KConfig>
@@ -30,6 +31,8 @@ class AbstractDailyWeatherForecast;
 
 Q_DECL_EXPORT int main(int argc, char *argv[])
 {
+    QIcon::setThemeName(QStringLiteral("breeze"));
+
     // We always NEED QApplication, since we use QtCharts
     QApplication app(argc, argv);
 
